packages/eslint-tool/README.md

```diff
@@ -0,0 +1,56 @@
+## @dotcom-reliability-kit/eslint-tool
+
+A prototype linting tool, specifically focussed on enhancing code quality and proactively catching errors/bugs before they make it into production. This is not a replacement for the [@financial-times/eslint-config-next](https://github.com/Financial-Times/eslint-config-next), but it can be installed alongside it or as an extension in it's config. This module is part of [FT.com Reliability Kit](https://github.com/Financial-Times/dotcom-reliability-kit#readme).
+
+- [Usage](#usage)
+  - [Static Code Analysis](#static-code-analysis)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Usage
+
+---
```

Comment:
**suggestion:** these [horizontal rules](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) after headings aren't necessary as they result in some invalid sectioning in the output HTML. They should used to denote a thematic break between paragraphs rather than to underline titles as screen readers will pick them up.

Reply:
Ohh thanks - this is helpful to know #TIL